components/CategoryHeadline.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+import { IconChevronDown, IconChevronUp } from "./Icons";
+import { PTagBold } from "./HtmlComponents";
+
+export default function CategoryHeadline(props) {
```

Comment:
Maybe you want to destructure your props here, keep it uniform across your whole app